message/validation/roundthresholds/round_thresholds.go

```diff
@@ -0,0 +1,87 @@
+package roundthresholds
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+
+	beaconprotocol "github.com/bloxapp/ssv/protocol/v2/blockchain/beacon"
+	"github.com/bloxapp/ssv/protocol/v2/qbft/roundtimer"
+)
+
+// Cache contains thresholds when each round finishes for each role.
+type Cache struct {
+	bn         beaconprotocol.BeaconNetwork
+	thresholds map[spectypes.BeaconRole][]time.Duration
+	mu         sync.Mutex
+}
```

Comment:
does this cache really needs to be mutex protected? I see Init is called once in the beginning and after that it only gets read. also it seems like that cache is not the right name for this in that case, maybe its a mapping or legend.

Reply:
Actually we can remove it, it's a leftover from a previous state of code. Ok, I'll rename it to mapping